resources/invoices.html

```diff
@@ -218,8 +218,8 @@ <h2 class="text-danger">Invoice expired!</h2>
 
       new QRCode(document.getElementById("invoice"), {
         text: `lightning:${invoice}`,
-        width: 256,
-        height: 256,
+        width: 512,
+        height: 512,
```

Comment:
Thanks! Please provide screenshots on how this will look. This will affect all other payment processors.

Reply:
https://github.com/Cameri/nostream/pull/192/files#diff-f69983e16ee9d68a0ef53895a6bbe177642cd1bf764de32b0822bd31ce3ad36dR116

I also hardcoded the height of the QR code to what it was before (the JS said 256; the browser showed it as 300px; no idea why)

Reply:
The underlying problem was that some LNbits instances encode routing hints in the invoice, so the 256 px were too few to show an image that can be scanned properly

Reply:
Desktop

![image](https://user-images.githubusercontent.com/8138838/218197786-8d26ffea-e691-4a3d-9b19-eb28424db2df.png)

Mobile

![image](https://user-images.githubusercontent.com/8138838/218197940-7a82676f-7342-4c4a-baae-76fc0743e42f.png)


Not sure what to do about mobile. Do you even want to show a QR there?

Reply:
@michael1011 were you able to address this?